micropub.php

```diff
@@ -532,11 +550,19 @@ private static function handle_authorize_error( $code, $msg ) {
 		static::respond( $code, $msg );
 	}
 
+	private static function default_post_status() {
+		return MICROPUB_DRAFT_MODE ? 'draft' : 'publish';
+	}
+
 	private static function post_status( $mf2 ) {
 		$props = $mf2['properties'];
 		// If both are not set immediately return
 		if ( ! isset( $props['post-status'] ) && ! isset( $props['visibility'] ) ) {
-			return get_option( 'micropub_default_post_status', MICROPUB_DRAFT_MODE ? 'draft' : 'publish' );
+			$status = get_option( 'micropub_default_post_status' );
```

Comment:
minor: consider keeping the second (default value) arg here (probably use `static:default_post_status()`, and then if we still get something unexpected below, log a warning, since that shouldn't happen?